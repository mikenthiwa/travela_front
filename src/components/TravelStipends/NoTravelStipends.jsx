import React from 'react';
import Icon from '../../images/no-document.svg';

const NoTravelStipends = () => (
  <div className="no-templates list-templates">
    <div className="content">
      <img src={Icon} alt="" />
      <p>
        No Travel Stipends have been created
      </p>
    </div>
  </div>
);
export default NoTravelStipends;
