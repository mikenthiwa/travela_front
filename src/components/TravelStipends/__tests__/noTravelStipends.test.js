import React from 'react';
import { mount, shallow } from 'enzyme';
import NoTravelStipends from '../NoTravelStipends';

describe('<NoTravelStipends />', () => {
  it('renders without crashing', () => {
    const wrapper = mount(<NoTravelStipends />);
    expect(wrapper).toMatchSnapshot();
  });
});


