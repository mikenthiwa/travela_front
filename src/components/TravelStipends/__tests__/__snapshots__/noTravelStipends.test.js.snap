// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoTravelStipends /> renders without crashing 1`] = `
<div
  className="no-templates list-templates"
>
  <div
    className="content"
  >
    <img
      alt=""
      src="no-document.svg"
    />
    <p>
      No Travel Stipends have been created
    </p>
  </div>
</div>
`;
